src/app/shared/form/builder/ds-dynamic-form-ui/relation-lookup-modal/relationship.effects.ts

```diff
@@ -161,11 +161,13 @@ export class RelationshipEffects {
   }
 
   private addRelationship(item1: Item, item2: Item, relationshipType: string, submissionId: string, nameVariant?: string) {
+    console.log('addRelationship 1');
     const type1: string = item1.firstMetadataValue('relationship.type');
     const type2: string = item2.firstMetadataValue('relationship.type');
     return this.relationshipTypeService.getRelationshipTypeByLabelAndTypes(relationshipType, type1, type2)
       .pipe(
         mergeMap((type: RelationshipType) => {
+            console.log('addRelationship 2', type.id, item2, item1, nameVariant);
```

Comment:
Same here... This looks to be accidental & can be removed

Reply:
Done